index.js

```diff
@@ -72,6 +79,21 @@ async function needsUpdate (cartfile, installedCartfile) {
   });
 }
 
+async function adjustFileSystem () {
+  let areDependenciesUpdated = false;
+  if (!await fs.hasAccess(`${BOOTSTRAP_PATH}/Resources`)) {
```

Comment:
I assume these two blocks can be done as one in a loop

Reply:
Done!